.github/workflows/assemble.yml

```diff
@@ -30,10 +43,19 @@ jobs:
           # Report success even if previous step failed (Docker on MacOS runner is very unstable)
           exit 0;
       - name: Run Gradle (assemble)
```

Comment:
Could we keep the runner command the same for all combinations by passing in the runtime?
   `./gradlew assemble --parallel --no-build-cache -PDISABLE_BUILD_CACHE -Druntime.java=${{matrix.java}}`

Reply:
Yeah, I think we could, will change that, thanks!